### PR TITLE
Chore: cleanup documentation

### LINKED ISSUE #5: Cleanup documentation
- [x] README needs to be improved
- [x] Codeready container need to be removed from docs
- [x] Spellchecks
- [x] Update docs to reflect examples dir (and other) having moved

### PR DESCRIPTION
**Pull request by hikarukin**
- Remove unused CRC section from documentation

Fixes #5

### CODE REVIEW

**Comment by hikarukin** [comment]
@devotional-phoenix-97 @portly-halicore-76 Please read through _at least_ the following files and review their **technical correctness** as I'm unsure what the current state of installation method and e2e testing is.

- /README.md
- /docs/development-guide/index.md
- /docs/development-guide/e2e-tests.md

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -33,21 +33,22 @@ We welcome many types of contributions including:
...
+- Errors and log messages that are displayed by the involved software;
+- Any other detail that might be useful;
 
 If you are trying to report a vulnerability, please refer to the
 [security policy](SECURITY.md).
 
 ## Developer notes:
-- Because of depency issues we decided to use a stub instead of importing all dependencies behind the original code of ArgoCD.
-  More info in internal/stubs/argocd/v1beta1
+- Because of dependency issues we decided to use a stub instead of importing all
```
Comment (portly-halicore-76) [inline CONTRIBUTING.md]: Do we need this here? Might be better suited in the developer guide.

File: README.md [hunk 2]
```diff
@@ -22,84 +25,43 @@ kubectl apply -f \
...
+Then copy `config/crd/bases/cpet.belastingdienst.nl_paas.yaml` to the `opr-paas-config` repo and distribute with ArgoCD from there.
 
 ## Description
-We want our developer teams to be able to create application environments with great ease and provide PaaS as an interface for this.
-Application environments consist of:
-- argocd (namespace, quota, argocd CR
-- ci (namespace, quota, tekton example pipelines and tasks, etc.)
-- SSO (namespace, quota, keycloak)
-- Grafana (namepace, quota, Grafana)
+We want our developer teams to be able to create application environments with
```
Comment (portly-halicore-76) [inline README.md]: Looks like more or less the same info as above in the goal part. Please combine.

File: README.md [hunk 3]
```diff
@@ -22,84 +25,43 @@ kubectl apply -f \
...
-It is adviced to copy it to a config repo and use that to maintain your own deployment.
+The quick start yaml file is a result from parts of the config folder, which is
+in this repo as an example only. It is advised to copy it to a config repo and use
+that to maintain your own deployment.
+
+When changing the crd, first run `make manifests` in the root of this repository.
 
-When changing the crd, first run `make manifests` in the root of this repo.
-Then copy config/crd/bases/cpet.belastingdienst.nl_paas.yaml to the opr-paas-config repo and dsitribute with ArgoCD from there
+Then copy `config/crd/bases/cpet.belastingdienst.nl_paas.yaml` to the `opr-paas-config` repo and distribute with ArgoCD from there.
```
Comment (portly-halicore-76) [inline README.md]: Config no longer exists, must point to manifests folder.

File: README.md [hunk 4]
```diff
@@ -22,84 +25,43 @@ kubectl apply -f \
...
-- a secret with a newly generated keypair used for
-- a deployment running the operator and a deployment running an encryption service
+
+- a namespace called `paas-system`;
+- 2 CRDs (`PaaS` and `PaasNs`);
+- a service account, role, role binding, cluster role and cluster role binding for
+  all permissions required by the operator;
+- a viewer & an editor cluster role for PaaS and PaasNs resources;
+- a configmap with all operator configuration options;
+- a secret with a newly generated key pair used for;
```
Comment (portly-halicore-76) [inline README.md]: It should never. User must generate those IMHO. Besides that, it's not the current situation.

File: README.md [hunk 5]
```diff
@@ -22,84 +25,43 @@ kubectl apply -f \
 ```
```
Comment (portly-halicore-76) [inline README.md:24]: The file above isn't available. We do have an install.yaml file in the manifests root. Might point to that file instead.

File: README.md [hunk 6]
```diff
@@ -22,84 +25,43 @@ kubectl apply -f \
...
+- argocd (namespace, quota, argocd CR);
+- ci (namespace, quota, tekton example pipelines and tasks, etc.);
+- SSO (namespace, quota, keycloak);
+- Grafana (namespace, quota, Grafana);
 
 ## Background information
 - [build-kubernetes-operator-six-steps](https://developers.redhat.com/articles/2021/09/07/build-kubernetes-operator-six-steps#setup_and_prerequisites)
 - [operator sdk installation instructions](https://sdk.operatorframework.io/docs/installation/)
 
-## Getting Started
```
Comment (portly-halicore-76) [inline README.md:54]: Nice

File: docs/development-guide/index.md [hunk 7]
```diff
@@ -85,21 +86,21 @@ Submit a pull request
...
+If you have written code for an improvement to Paas or a bug fix,
 please follow this procedure to submit a pull request:
 
-1. [Create a fork](development-environment/index.md#forking-the-repository) of PAAS
+1. [Create a fork](development-environment/index.md#forking-the-repository) of Paas
 2. Self-assign the ticket and begin working on it in the forked project. Move
    the ticket to `Analysis` or `In Development` phase of
-   [PAAS operator development](https://github.com/orgs/belastingdienst/projects/2)
+   [Paas operator development](https://github.com/orgs/belastingdienst/projects/2)
 3. [Run the e2e tests in the forked repository](e2e_testing_environment/README.md#running-e2e-tests-on-a-fork-of-the-repository)
```
Comment (portly-halicore-76) [inline docs/development-guide/index.md]: This doesn't exist.

**Comment by portly-halicore-76** [comment]
I've included updating the e2e test description in PR. https://github.com/belastingdienst/opr-paas/pull/23

File: README.md (hunk 4 above)
Comment (hikarukin) [reply README.md]: Removed line about secret being generated

File: docs/development-guide/index.md (hunk 7 above)
Comment (hikarukin) [reply docs/development-guide/index.md]: Removed

File: docs/core_concepts/sshsecrets.md [hunk 8]
```diff
@@ -76,24 +102,28 @@ metadata:
 spec:
   # Specifying a sshSecret for all capability- and functional- namespaces
   sshSecrets:
-    "ssh://git@github.com/belastingdienst/paas.git": >-
+    "ssh://git@github.com/belastingdienst/Paas.git": >-
```
Comment (portly-halicore-76) [inline docs/core_concepts/sshsecrets.md]: even thought it's an example, this is a cmd a replace thingy?

File: docs/core_concepts/sshsecrets.md [hunk 9]
```diff
@@ -103,7 +133,7 @@ metadata:
   name: my-ns
   namespace: my-paas-argocd
 spec:
-  paas: my-paas
+  Paas: my-paas
```
Comment (portly-halicore-76) [inline docs/core_concepts/sshsecrets.md]: Idem

File: docs/core_concepts/sshsecrets.md [hunk 10]
```diff
@@ -76,24 +102,28 @@ metadata:
...
+
+The PaasNs controller is the one to manage the secrets in the Paas namespaces a
+defined in the PaasNs (either manually created or managed by the Paas controller).
 
-The PaasNs controller is the one to manage the secrets in the PaaS namespaces as defined in the PaasNs (either manually created or managed by the PaaS controller).
-The PaasNs controller will update ssh secrets in the namespace if the sshSecret value is changed in the PaasNs resource.
-But when the key changes (e.a. `ssh://git@github.com/belastingdienst/paas.git` in the example below), the original ssh secret is not removed.
+The PaasNs controller will update SSH secrets in the namespace if the `sshSecret`
+value is changed in the PaasNs resource. However, when the key changes
+(e.a. `ssh://git@github.com/belastingdienst/Paas.git` in the example below), the
```
Comment (portly-halicore-76) [inline docs/core_concepts/sshsecrets.md]: Idem

File: docs/development-guide/index.md [hunk 11]
```diff
@@ -58,21 +59,21 @@ TODO:
...
 
-If you want to test or evaluate the latest development snapshot of
-PAAS before the next official patch release, you can simply run:
+If you want to test or evaluate the latest development snapshot of Paas before
+the next official patch release, you can simply run:
 
 ```sh
 kubectl apply -f \
-  https://raw.githubusercontent.com/belastingdienst/artifacts/main/manifests/paas-operator-manifest.yaml
+  https://raw.githubusercontent.com/belastingdienst/artifacts/main/manifests/Paas-operator-manifest.yaml
```
Comment (portly-halicore-76) [inline docs/development-guide/index.md]: Have we changed this somewhere else just now? Isn't that duplicate and not right in this case.

**Review by portly-halicore-76 — APPROVED** [review]
Gr8 update for the docs.

**Pull request merged**